README.md

```diff
@@ -36,7 +36,6 @@ Please also be aware that it will be required for development team to review cha
 
     ```
     cd _src
-    bin/sculpin install
```

Comment:
actually we should, you're right. but I wonder if we need to do this every time? maybe we should prep a "initialize" script that would prevent install being triggered every time - what do you think?